src/commcare_cloud/ansible/deploy_redis.yml

```diff
@@ -0,0 +1,63 @@
+- name: Redis
+  hosts: redis
+  become: true
+  roles:
+    - {role: redis, tags: 'redis'}
+
+- name: configure the master redis server
```

Comment:
This redis/master/sentinel setup is to be run for only ICDS env, right? Are we rolling this out to all envs?

Reply:
only for ICDS yes. This will only run on hosts in the `redis_master` group which no other envs have. Is there a better way you know of to segregate this for only ICDS?

Reply:
I just got confused with the playbook name, that's fine then. I would probably create a separate `setup_redis_ha.yml` and call that from here when the env is configured for Redis HA (similar to the [setup_cluster](https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/roles/redis/tasks/main.yml#L4).)

Reply:
The example you give is basically exactly the same. It includes the tasks if there are hosts in the `redis_cluster_master` group